Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -81,7 +81,10 @@ Describe "BqJob-Query" {
     It "should query out of a pre-loaded table" {
```

Comment:
This is not related to this CR but it seems we don't have a test for -Synchronous switch of Start-BqJob? Maybe you should add one.